CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [4.0.33](https://github.com/puppetlabs/facter/tree/4.0.33) (2020-08-05)
+
+[Full Changelog](https://github.com/puppetlabs/facter/compare/4.0.32...4.0.33)
+
+### Added
+- \(FACT-2040\) Added solaris memory resolver [\#2040](https://github.com/puppetlabs/facter/pull/2040) ([sebastian-miclea](https://github.com/sebastian-miclea))
+
+### Fixed
+- \(FACT-2735\) When no hypervisor is detected, return nil [\#2004](https://github.com/puppetlabs/facter/pull/2004) ([IrimieBogdan](https://github.com/IrimieBogdan))
```

Comment:
Should be the PR title not the commit message

Reply:
Fixed